app/controllers/posts_controller.rb

```diff
@@ -4,7 +4,15 @@ class PostsController < ApplicationController
   before_action :correct_user, only: %i(edit update)
 
   def index
-    @posts = Post.order(created_at: :desc).page(params[:page]).per Settings.posts.page.max
+    @topics = Topic.all
```

Comment:
add scope sort
example
- in Topic model file
`scope :lastest, ->{order created_at: :desc}`

- in here
change code ` @topics = Topic.all` to ` @topics = Topic.lastest`